packages/components/src/components/common/table/edit/edit.tsx

```diff
@@ -36,13 +36,13 @@ const InputCell = <TData,>({ cell, className, ...props }: InputCellProps<TData>)
 };
 InputCell.displayName = 'InputCell';
 
-type SelectCellProps<TData> = SimpleSelectProps & {
+type SelectCellProps<TData> = BasicSelectProps & {
   cell: CellContext<TData, string>;
 };
 
 const SelectCell = <TData,>({ cell, className, ...props }: SelectCellProps<TData>) => {
   const { value, updateValue, className: editCell } = useEditCell(cell);
-  return <SimpleSelect value={value} onValueChange={updateValue} className={cn(editCell, className)} {...props} />;
+  return <BasicSelect value={value} onValueChange={updateValue} className={cn(editCell, className)} {...props} />;
```

Comment:
changed also to BasicSelect (SimpleSelect is deprecated as I streamlined all basic component implementations with a Basic prefix)